docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -7,21 +7,15 @@
     "tags": []
```

Comment:
<p>Suggestion for the title <code>Tracking Progress in {mod}ott.solvers</code></p><p>For consistency, please add <code>,</code> after each list item. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>